package.json

```diff
@@ -59,5 +59,8 @@
     "nyc": "^15.1.0",
     "rollup": "^2.53.2",
     "tsd": "^0.31.2"
+  },
+  "dependencies": {
+    "pac-proxy-agent": "^7.0.2"
```

Comment:
This should be in package.json of core package

Reply:
If we include it inside module itself, then it should be inside client module, right?

Reply:
@ninadbstack for now I've added in client one, let me know if that is incorrect or something

Reply:
Correct, my bad, it should be in client.